src/test/java/com/microsoft/sqlserver/jdbc/AlwaysEncrypted/AESetup.java

```diff
@@ -283,7 +280,7 @@ public static void cleanUp() throws Exception {
      * @param lookupValue
      * @throws IOException
      */
-    private static void readFromFile(String inputFile, String lookupValue) throws IOException {
+    protected static void readFromFile(String inputFile, String lookupValue) throws IOException {
```

Comment:
I don't think this method needs to be protected, it's not used outside this file?

Reply:
you're right it's not, I changed this at some point but forgot to change it back. thx